Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -1,594 +1,344 @@
 {
-  background-color: #F0F0F0;
+  border: var(--remotion-themed-border-width) solid transparent;
+  border-radius: 2px;
+  padding: var(--remotion-themed-spacing-half);
+  margin: 0 var(--remotion-themed-spacing-half);
+  background-color: var(--remotion-themed-control-background);
 }
 
-/*  Style applied to the cells in the BocList's selected odd and even data rows. */
-tr.bocListDataRow.selected td.bocListDataCell
+div.bocList div.bocListTableBlock div.bocListNavigator a[href]
 {
-  background-color: #C5E0FC;
+  border-color: var(--color-standard-border);
 }
 
-
-/*  Style applied to the cell in the BocList's data row that contains the index. */
-th.bocListTitleCellIndex
+div.bocList div.bocListTableBlock div.bocListNavigator a[href]:hover
 {
-  text-align: center;
+  border-color: var(--color-primary);
+  background-color: var(--color-highlight);
 }
 
-/*  Style applied to the cell in the BocList's data row that contains the index. */
-td.bocListDataCellIndex
+div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 {
-  text-align: center;
+  filter: opacity(0.5);
 }
+/*endregion*/
 
-/*  Style applied to the non-anchor content in the data rows.  */
-span.bocListContent,
-label.bocListContent
+/*region bocListMenuBlock*/
+div.bocList div.bocListMenuBlock
 {
-  padding: 2px;
+  grid-column: 2;
+  grid-row: 1;
+  background-color: var(--boclist-listmenu-background-color);
 }
 
-/*  Style applied to the anchor tags in BocList's data rows.  */
-tr.bocListDataRow td.bocListDataCell a
+div.bocList div.bocListMenuBlock > div
 {
-  display: inline-block;
-  padding: 1px;
-  border: 1px solid transparent;
+  margin: var(--remotion-themed-spacing);
+  display: flex;
+  flex-direction: column;
+  gap: var(--remotion-themed-spacing);
 }
 
-/*  Style applied to the anchor tags in BocList's selected data rows.  */
-tr.bocListDataRow.selected td.bocListDataCell a
+div.bocList div.bocListMenuBlock div.bocListAvailableViewsList
 {
+  display: flex;
+  align-items: center;
 }
 
-/*  Style applied to the anchor tags during hovering in BocList's data rows.  */
-tr.bocListDataRow td.bocListDataCell a:focus,
-tr.bocListDataRow td.bocListDataCell a:hover
+div.bocList div.bocListMenuBlock div.bocListAvailableViewsList select.bocListAvailableViewsListDropDownList
 {
-  color: Black;
-  border: 1px solid #7CB2E6;
-  background-color: #E5F0FC;
-  outline: none;
+  flex-grow: 1;
 }
 
-tr.bocListDataRow td.bocListDataCell a:active
+div.bocList div.bocListMenuBlock .DropDownMenuContainer
 {
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
+  width: 100%;
 }
+/*endregion*/
 
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie tr.bocListDataRow td.bocListDataCell > a
+/*region Table head*/
+div.bocList div.bocListTableBlock div.bocListTableContainer thead.bocListTableHead
 {
-  position: relative;
+  border-bottom: 1px solid var(--color-standard-border);
```

Comment:
```suggestion
  border-top: 1px solid var(--color-standard-border);
```
![image](https://user-images.githubusercontent.com/6972548/133264300-c22f3c57-4e5a-4b9e-83e5-b0bbfb8a4b8d.png)